bin/loklakinit.py

```diff
@@ -9,151 +10,247 @@
   app_json["author"] = author
 
-  print json.dumps(app_json, indent=2)
+  # keep taking inputs from user until he confirms
+  while True :
+    app_name = raw_input("name (" + app_json["name"] + ") : ")
+    if app_name :
+      app_json["name"] = app_name
+
+    app_context = raw_input("@context (" + app_json["@context"] + ") : ")
+    if app_context :
+      app_json["@context"] = app_context
+
+    app_type = raw_input("@type (" + app_json["@type"] + ") : ")
+    if app_type :
+      app_json["@type"] = app_type
+
+    app_permissions = raw_input("permissions (" + app_json["permissions"] + ") : ")
+    if app_permissions :
+      app_json["permissions"] = app_permissions.split(",")
+
+    version = raw_input("version (" + app_json["version"] + ") : ")
+    if version :
+      app_json["version"] = version
+
+    app_headline = raw_input("headline (" + app_json["headline"] + ") : ")
+    if app_headline :
+      app_json["headline"] = app_headline
+      app_json["alternativeHeadline"] = app_headline
+
+    app_alternative_headline = raw_input("alternative headline (" + app_json["alternativeHeadline"] + ") : ")
+    if app_alternative_headline :
+      app_json["alternativeHeadline"] = app_alternative_headline
+
+    app_category = raw_input("application category (" + app_json["applicationCategory"] + ") : ")
+    if app_category :
+      app_json["applicationCategory"] = app_category
+
+    app_sub_category = raw_input("sub category : ")
+    if app_sub_category :
+      app_json["applicationSubCategory"] = app_sub_category
+
+    app_os = raw_input("application operating system (" + app_json["operatingSystem"] + ") : ")
+    if app_os :
+      app_json["operatingSystem"] = app_os
+
+    app_promo_image = raw_input("promo image (" + app_json["promoImage"] + ") : ")
+    if app_promo_image :
+      app_json["promoImage"] = app_promo_image
+
+    app_images = raw_input("app preview images (values can be separted by comma) : ")
+    if app_images :
+      app_json["appImages"] = app_images.split(",")
+
+    app_description = raw_input("one line description : ")
+    if app_description :
+      app_json["oneLineDescription"] = app_description
+
+    tech_stack = raw_input("technology stack used (items can be separated by comma) : ")
+    if tech_stack :
+      app_json["techStack"] =  tech_stack.split(",")
+
+    app_source = raw_input("app source : ")
+    if app_source :
+      app_json["appSource"] = app_source
+
+    author["name"] = raw_input("author name : ")
+
+    author_type = raw_input("author type (" + author["@type"] + ") : ")
+    if author_type :
+      author["@type"] = author_type
+
+    author["email"] = raw_input("email : ")
+
+    author["url"] = raw_input("url : ")
+    author["sameAs"] = author["url"]
+
+    author["sameAs"] = raw_input("same as : ")
+
+    app_json["author"] = author
+
+    app_json["contributors"] = [{"name": author["name"], "url": author["url"]}]
+
+    today = datetime.datetime.now().date();
```

Comment:
@djmgit here only date is printed using `today_date` variable, what do you think about printing complete datetime? Since this is `loklakinit.py`, this shall be printed.